docs/docs/fallback-handoff.mdx

```diff
@@ -283,6 +285,49 @@ rules:
   - active_loop: action_two_stage_fallback
 ```
 
+### 4. Defining an ultimate fallback action
+
+To define the bot's response when the user denies the rephrased intent, define the response `utter_default`:
+
+```yaml-rasa title="domain.yml"
+responses:
+  utter_default:
+  - text: I'm sorry, I can't help you.
+```
+
+Or, you can customize `action_default_fallback` for more complex behavior by writing a [Custom Action](./actions.mdx#custom-actions). 
+For example, if you want the bot to call a human and stop interacting with the user:
+
+```python title="actions.py"
+from typing import Any, Dict, List, Text
+
+from rasa_sdk import Action, Tracker
+from rasa_sdk.events import UserUtteranceReverted
+from rasa_sdk.executor import CollectingDispatcher
+
+class ActionDefaultFallback(Action):
+    def name(self) -> Text:
+        return "action_default_fallback"
+
+    def run(
+        self,
+        dispatcher: CollectingDispatcher,
+        tracker: Tracker,
+        domain: Dict[Text, Any],
+    ) -> List[Dict[Text, Any]]:
+
+        # tell the user they are being passed to a customer service agent
+        dispatcher.utter_message(text="I am passing you to a human...")
+        
+        # assume there's a function to call customer service
+        # pass the tracker so that the agent has a record of the converstation between the user
+        # and the bot for context
+        call_customer_service(tracker)
+     
+        # pause the tracker so that the bot stops responding to user input
+        return [ConversationPaused()]
+```
+
```

Comment:
@melindaloubser1 what do you think of this mock handoff?

Reply:
That's great!